File: wellness-core-backend/src/configs/envs.ts
import * as dotenv from 'dotenv';

// eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
dotenv.config();

export const CORE_API_BASE_URL =
  process.env.CORE_API_BASE_URL || 'http://localhost:3000';

export const DB_HOST = process.env.DB_HOST || '18.10.1.103';
export const DB_PORT = parseInt(process.env.DB_PORT || '5433', 10);
export const DB_NAME = process.env.DB_NAME || 'wellness_db';
export const DB_USERNAME = process.env.DB_USERNAME || 'postgres';
export const DB_PASSWORD = process.env.DB_PASSWORD || 'postgres';

File: apps/mobile-app/lib/configs/envs.dart
import 'package:flutter_dotenv/flutter_dotenv.dart';

class Envs {
  static const String defaultCoreApiBaseUrl = 'http://localhost:9100';

  static String get coreApiBaseUrl {
    try {
      return dotenv.env['CORE_API_BASE_URL'] ?? defaultCoreApiBaseUrl;
    } catch (_) {
      return defaultCoreApiBaseUrl;
    }
  }
}
